docker/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM maven:3.6.3-jdk-8 AS builder
```

Comment:
A docker file for a common repo? why? Shouldn't it be enough to add it as a maven dependency in the other repos?

Reply:
It is not enough. Without an image to store the commons artifact I would need to clone the common repository in building the images for the other services.